<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>1018</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>DentalAnalyzer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBoxOutput">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Outputs</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="2" column="0">
          <widget class="QTextBrowser" name="textBrowserOutput">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Arial','Arial','Arial';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="2">
          <widget class="QGroupBox" name="GroupBoxAnalyze">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="title">
            <string>Inputs</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0">
            <item row="7" column="1">
             <widget class="QLineEdit" name="lineEditStudentMarginPoints">
              <property name="focusPolicy">
               <enum>Qt::ClickFocus</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="placeholderText">
               <string>No File Specified</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEditStudentFolder">
              <property name="focusPolicy">
               <enum>Qt::ClickFocus</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="placeholderText">
               <string>No Folder Specified</string>
              </property>
             </widget>
            </item>
            <item row="15" column="1">
             <widget class="QLineEdit" name="lineEditStudentMidpoint">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="placeholderText">
               <string>No File Specified</string>
              </property>
             </widget>
            </item>
            <item row="7" column="2">
             <widget class="QPushButton" name="pushButtonStudentMarginPoints">
              <property name="text">
               <string>Open File</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_9">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Student Name</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Student Axial Points</string>
              </property>
             </widget>
            </item>
            <item row="14" column="1">
             <widget class="QLineEdit" name="lineEditStudentCenter">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="placeholderText">
               <string>No File Specified</string>
              </property>
             </widget>
            </item>
            <item row="11" column="2">
             <widget class="QPushButton" name="pushButtonOriginalModel">
              <property name="text">
               <string>Open File</string>
              </property>
             </widget>
            </item>
            <item row="11" column="1">
             <widget class="QLineEdit" name="lineEditOriginalModel">
              <property name="text">
               <string/>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="placeholderText">
               <string>No File Specified</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Student Folder</string>
              </property>
             </widget>
            </item>
            <item row="14" column="0">
             <widget class="QLabel" name="labelStudentCenter">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Student Model Center</string>
              </property>
             </widget>
            </item>
            <item row="11" column="0">
             <widget class="QLabel" name="label_7">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Original Model</string>
              </property>
             </widget>
            </item>
            <item row="8" column="2">
             <widget class="QPushButton" name="pushButtonStudentAxialPoints">
              <property name="text">
               <string>Open File</string>
              </property>
             </widget>
            </item>
            <item row="15" column="2">
             <widget class="QPushButton" name="pushButtonStudentMidpoint">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Open File</string>
              </property>
             </widget>
            </item>
            <item row="25" column="0" colspan="4">
             <widget class="QPushButton" name="pushButtonAnalyze">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>15</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Analyze</string>
              </property>
             </widget>
            </item>
            <item row="14" column="2">
             <widget class="QPushButton" name="pushButtonStudentCenter">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Open File</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Student Margin Points</string>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Student Occlusal Points</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEditStudentName">
              <property name="placeholderText">
               <string>Student Name</string>
              </property>
             </widget>
            </item>
            <item row="10" column="2">
             <widget class="QPushButton" name="pushButtonStudentGingivaPoints">
              <property name="text">
               <string>Open File</string>
              </property>
             </widget>
            </item>
            <item row="15" column="0">
             <widget class="QLabel" name="labelStudentMidpoint">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Student Model Midpoint</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QPushButton" name="pushButtonStudentFolder">
              <property name="text">
               <string>Open Folder</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <property name="autoRepeat">
               <bool>false</bool>
              </property>
              <property name="default">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="lineEditStudentModel">
              <property name="focusPolicy">
               <enum>Qt::ClickFocus</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="placeholderText">
               <string>No File Specified</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Student Model</string>
              </property>
             </widget>
            </item>
            <item row="9" column="1">
             <widget class="QLineEdit" name="lineEditStudentOcclusalPoints">
              <property name="focusPolicy">
               <enum>Qt::ClickFocus</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="placeholderText">
               <string>No File Specified</string>
              </property>
             </widget>
            </item>
            <item row="10" column="1">
             <widget class="QLineEdit" name="lineEditStudentGingivaPoints">
              <property name="focusPolicy">
               <enum>Qt::ClickFocus</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="placeholderText">
               <string>No File Specified</string>
              </property>
             </widget>
            </item>
            <item row="13" column="0">
             <widget class="QCheckBox" name="checkBoxDivision">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Enable Region Division</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="QLineEdit" name="lineEditStudentAxialPoints">
              <property name="focusPolicy">
               <enum>Qt::ClickFocus</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="placeholderText">
               <string>No File Specified</string>
              </property>
             </widget>
            </item>
            <item row="9" column="2">
             <widget class="QPushButton" name="pushButtonStudentOcclusalPoints">
              <property name="text">
               <string>Open File</string>
              </property>
             </widget>
            </item>
            <item row="10" column="0">
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="text">
               <string>Student Ginviga Points</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QPushButton" name="pushButtonStudentModel">
              <property name="text">
               <string>Open File</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBoxFeedback">
        <property name="title">
         <string>Feedback</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0" colspan="3">
          <widget class="QTextEdit" name="textEdit">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QPushButton" name="pushButtonSaveFeedback">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Save Feedback</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>849</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
